okhttp/src/main/java/okhttp3/internal/http2/Http2Connection.java

```diff
@@ -320,32 +321,40 @@ void addBytesToWriteWindow(long delta) {
   }
 
   void writeSynResetLater(final int streamId, final ErrorCode errorCode) {
-    writerExecutor.execute(new NamedRunnable("OkHttp %s stream %d", hostname, streamId) {
-      @Override public void execute() {
-        try {
-          writeSynReset(streamId, errorCode);
-        } catch (IOException e) {
-          failConnection();
+    try {
+      writerExecutor.execute(new NamedRunnable("OkHttp %s stream %d", hostname, streamId) {
+        @Override public void execute() {
+          try {
+            writeSynReset(streamId, errorCode);
+          } catch (IOException e) {
+            failConnection();
+          }
         }
-      }
-    });
+      });
+    } catch (RejectedExecutionException ignored) {
```

Comment:
GitHub isn’t doing a great job showing this diff. It’s just adding this catch clause everywhere we enqueue a job on an executor that might have been shutdown.